src/components/AdjustVaultSummary.tsx

```diff
@@ -0,0 +1,146 @@
+import clsx from 'clsx';
+import type { Amount } from '@agoric/ertp/src/types';
+import { useAtomValue } from 'jotai';
+import { displayFunctionsAtom } from 'store/app';
+import { useVaultStore } from 'store/vaults';
+import { makeRatioFromAmounts } from '@agoric/zoe/src/contractSupport';
+
+type TableRowProps = {
+  left: string;
+  right: string;
+};
+
+const TableRow = ({ left, right }: TableRowProps) => {
+  return (
+    <tr className="text-[13px] leading-[21px]">
+      <td className="text-[#A3A5B9] text-left pr-2">{left}</td>
+      <td className="text-right text-[#666980] font-black">{right}</td>
+    </tr>
+  );
+};
+
+type TableRowWithArrowProps = {
+  label: string;
+  left: string;
+  right: string;
+};
+
+const TableRowWithArrow = ({ label, left, right }: TableRowWithArrowProps) => {
+  return (
+    <tr className="text-[13px] leading-[21px]">
+      <td className="text-[#A3A5B9] text-left">{label}</td>
+      <td className="text-[#666980] font-black px-2">{left}</td>
+      <td className="text-[#666980] font-black px-2">&#8594;</td>
+      <td className="text-right text-[#666980] font-black">{right}</td>
+    </tr>
+  );
+};
+
+type Props = {
+  locked: Amount<'nat'>;
+  debt: Amount<'nat'>;
+  lockedValue: Amount<'nat'>;
+  managerId: string;
+};
+
+const AdjustVaultSummary = ({
+  locked,
+  debt,
+  lockedValue,
+  managerId,
+}: Props) => {
+  const canMakeOffer = true;
+  const adjustButtonLabel = 'Make Offer';
+
+  const { governedParams } = useVaultStore(state => ({
+    governedParams: state.vaultGovernedParams,
+  }));
+
+  const params = governedParams.get(managerId);
+
+  const collateralizationRatio = makeRatioFromAmounts(lockedValue, debt);
+
+  const { displayAmount, displayBrandPetname, displayPercent } =
+    useAtomValue(displayFunctionsAtom) ?? {};
```

Comment:
if you do make defaults, they'd apply here too.

given the composable nature of Jotai, it could be another atom that returns existing `displayFuncions` if they're available and loading ones if they're not. Though now that I think about it why wouldn't we always want display functions to be available and usable? These UI components shouldn't have to condition on the state of that atom.

Reply:
Yes we want to show loading states on certain components depending on if we've loaded display functions or not (among other data). I'd like to maintain the nullability of `displayFunctionsAtom` for that purpose. For these components we just don't care about showing a loading state because we know we'll never get here without having loaded a vault summary card first, which requires display functions.